qwinto/board_logic/board.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+class Board:
+	def __init__(self):
+		self.arr = np.zeros(3,12)
+		""" layout array 
+			1 = not part of the board
+			-1 = the pentagon symbol on board, the bonus points for a full column"""
```

Comment:
Triple quotes denote a docstring. I believe it is the convention to put the docstring in the first line of the function. Otherwise use line comments (`#`).